src/cmd-build

```diff
@@ -231,7 +231,13 @@ composejson=${PWD}/tmp/compose.json
 # Put this under tmprepo so it gets automatically chown'ed if needed
 lockfile_out=${tmprepo}/tmp/manifest-lock.generated.${basearch}.json
 prepare_compose_overlays
-# --cache-only is here since `fetch` is a separate verb.
+# --cache-only is here since `fetch` is a separate verb; however, we do a fetch
+# explicitly if there appears to be no cache, in order to allow people to just
+# do `cosa build` on a clean dir.
```

Comment:
What do want the local developer workflow to look like? The setup I have doesn't take the `runvm` route so it doesn't bother me but if a user's setup does take that route this solution only solves the problem for them the first time they run it.

Would it be better to add a `--nonet` option to `cosa build` and just use that in the pipeline?

Reply:
> if a user's setup does take that route this solution only solves the problem for them the first time they run it.

No - what's going on here is that rpm-ostree if passed `--cache-only` will still fetch if it doesn't have metadata, which IMO is a bug.   But it's one we were relying on.

But with this it's simple to explain; if you want new content, you `cosa fetch`, which will have networking.  Otherwise we shouldn't hit the network.